src/backend/wsl.ts

```diff
@@ -1119,6 +1119,18 @@ export default class WSLBackend extends events.EventEmitter implements K8s.Kuber
     await this.progressTracker.action(description, 10, async() => {
       try {
         this.setState(K8s.State.STARTING);
+
+        // Register a new tunnel for RD Guest Agent
+        if (process.platform === 'win32') {
+          this.vtun.addTunnel({
```

Comment:
This doesn't _need_ to be run on every call to `start` (just once, possible around the constructor), right?